st3/sublime_lib/__init__.py

```diff
@@ -2,3 +2,5 @@
 from .settings_dict import NamedSettingsDict  # noqa: F401
 from .view_stream import ViewStream  # noqa: F401
 from .output_panel import OutputPanel  # noqa: F401
+from .window_utils import new_window  # noqa: F401
+from .window_utils import close_window  # noqa: F401
```

Comment:
Why not have these on a single line?

Reply:
For some reason, I thought that flake would complain. It does not.